src/components/Form.js

```diff
@@ -11,34 +12,32 @@ const Form = ({
   itemToEdit,
   setFilteredTodos,
   setInputText,
-  inputText
+  inputText,
 }) => {
   //State variables
   const [status, setStatus] = useState('all');
-  //Functions
 
-    //use effect
-    useEffect(() => {
-      filterHandler();
-    }, [todos, status]);
+  //Hook usage
+  useEffect(() => {
+    filterHandler();
+  }, [todos, status]);
 
-    //Functions
-    const filterHandler = () => {
-      switch (status) {
-        case 'completed':
-          setFilteredTodos(todos.filter(todo => todo.completed === true));
-          break;
-        case 'uncompleted':
-          setFilteredTodos(todos.filter(todo => todo.completed === false));
-          break;
-        case 'progress':
-          setFilteredTodos(todos.filter(todo => todo.progress === true));
-          break;
-        default:
-          setFilteredTodos(todos);
-          break;
-      }
-    };
+  const filterHandler = () => {
+    switch (status) {
+      case 'completed':
+        setFilteredTodos(todos.filter(todo => todo.completed));
+        break;
+      case 'uncompleted':
+        setFilteredTodos(todos.filter(todo => todo.completed === false));
```

Comment:
Explicit boolean comparison, pls try using logical operators (logical not suits here)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_NOT

Reply:
fixed